src/guiapi/include/MItem_tools.hpp

```diff
@@ -0,0 +1,197 @@
+/*****************************************************************************/
+//menu items running tools
+#pragma once
+#include "WindowMenuItems.hpp"
+#pragma pack(push, 1)
+
+class MI_WIZARD : public WI_LABEL_t {
+    static constexpr const char *const label = "Wizard";
```

Comment:
This will be very hard to localize (translate), because the text here is a static const char * but the real translation will be done in IWindowMenuItem::printText() when it calls render_text_align().
Thus all these strings will have to be tagged with some other macro than _() just to force gettext to extract this string from the source code.
Then, in render_text_align there will be the final gettext() call to get the localized string